MeshLib/Elements/Cell.h

```diff
@@ -26,29 +25,13 @@ namespace MeshLib {
 class Cell : public Element
 {
 public:
-	/// Constant: Dimension of this mesh element
-	static const unsigned dimension = 3;
-
-	/// Returns the length, area or volume of a 1D, 2D or 3D element
-	double getContent() const { return _volume; }
-
-	/// Get dimension of the mesh element.
-	unsigned getDimension() const { return dimension; }
-
 	/// Get the volume of this 3d element.
-	virtual double getVolume() const { return _volume; }
+	virtual double getVolume() const { return getContent(); }
```

Comment:
Are you suggesting renaming getVolume() with getCellSize()? If so, why? What we return here is volume of a cell (which is 3D).